charts/terraform-cloud/templates/_helpers.tpl

```
@@ -80,7 +80,7 @@ app.mintel.com/region: {{ .Values.global.clusterRegion }}
 {{/* ternary and hasKey functions are used instead of defaults below due to https://github.com/helm/helm/issues/3308 */}}
 app.mintel.com/terraform-allow-destroy: {{ hasKey .InstanceCfg "workspaceAllowDestroy" | ternary .InstanceCfg.workspaceAllowDestroy (include "mintel_common.terraform_cloud.allow_destroy_default" .) | quote }}
 app.mintel.com/terraform-owner: {{ default .Global.owner .InstanceCfg.workspaceOwner }}
-app.mintel.com/terraform-cloud-tags: {{ default (include "mintel_common.terraform_cloud.tags" .) .InstanceCfg.workspaceTags }}
+app.mintel.com/terraform-cloud-tags: {{ default (include "mintel_common.terraform_cloud.tags" .) .InstanceCfg.workspaceTags | trunc 63 | quote}}
```

Should we avoid truncating this? Truncating tags would end up effectively breaking the tags. 

Should we let this fail instead?

---

That's a good point. I put that as a safeguard basically and if it does truncate, it'll be module name (hence why it's last) which we'd still be able to make out. Maybe the best option is to remove it and make an OPA rule